### PR TITLE
[Task] CS Fixer for Micro Optimiziation

### PR DESCRIPTION
**Pull request by mattamon**
See title

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -72,6 +72,7 @@
     'ternary_operator_spaces'             => true,
     'trailing_comma_in_multiline'         => true,
     'whitespace_after_comma_in_array'     => true,
+    'import_functions'                    => true,
```
Comment (jdreesen) [inline .php-cs-fixer.dist.php]: See: https://github.com/pimcore/studio-backend-bundle/issues/200#issuecomment-2197184449

Comment (mattamon) [reply .php-cs-fixer.dist.php]: Thanks for the comment! I will add it!

Comment (jdreesen) [reply .php-cs-fixer.dist.php]: You can still use `'global_namespace_import' => ['import_functions' => true],` if you prefer imports to backslashes before every function call.

Comment (mattamon) [reply .php-cs-fixer.dist.php]: yeah. I just readded it :) Thanks for the hint though. With all the new bundles we try to rather do use statements.

Comment (jdreesen) [reply .php-cs-fixer.dist.php]: There's also [`ordered_imports`](https://mlocati.github.io/php-cs-fixer-configurator/#version:3.59|fixer:ordered_imports) if you want to separate/group class/const/function imports.

Comment (mattamon) [reply .php-cs-fixer.dist.php]: even better, yeah i find it not so nice if they are mixed. thank you very much!

**Pull request merged**